src/merge.py

```diff
@@ -16,7 +15,16 @@ def merge(x: list[int], y: list[int]) -> list[int]:
     # FIXME: fill out the loop so you merge the lists
     # until one of them is empty
     while i < len(x) and j < len(y):
-        break  # FIXME: you shouldn't just break here
+        if x[i] <= y[j]:
+            z.append(x[i]) 
+            i += 1
+        if y[j] < x[i]: 
+            z.append(y[j])
+            j += 1
+        if len(z) == (len(x) + len(y)):
+            pass
```

Comment:
This will never do anything. The code says that if the condition is true, it shouldn't do anything, and if it is false it should continue to the code after the `if`-statement (without doing anything)